website/docs/r/oracledatabase_cloud_vm_cluster.html.markdown

```diff
@@ -0,0 +1,165 @@
+---
+subcategory: "App Service"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_oracledatabase_cloud_vm_cluster"
+description: |-
+  Manages a Cloud VM Cluster.
+---
+
+# azurerm_oracledatabase_cloud_vm_cluster
+
+Manages a Cloud VM Cluster.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_oracledatabase_exadata_infrastructure" "example" {
+  name                = "example-exadata-infrastructure"
+  display_name        = "example-exadata-infrastructure"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  shape               = "Exadata.X9M"
+  storage_count       = "3"
+  compute_count       = "2"
+  zones = ["3"]
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-virtual-network"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes      = ["10.0.1.0/24"]
+
+  delegation {
+    name = "delegation"
+
+    service_delegation {
+      actions = [
+        "Microsoft.Network/networkinterfaces/*",
+        "Microsoft.Network/virtualNetworks/subnets/join/action",
+      ]
+      name = "Oracle.Database/networkAttachments"
+    }
+  }
+}
+
+data "azurerm_oracledatabase_db_servers" "example" {
+  resource_group_name = azurerm_resource_group.example.name
+  cloud_exadata_infrastructure_name = azurerm_oracledatabase_exadata_infrastructure.example.name
+}
+
+resource "azurerm_oracledatabase_cloud_vm_cluster" "example" {
+  name = "example-cloud-vm-cluster"
+  resource_group_name = azurerm_resource_group.example.name
+  location = azurerm_resource_group.example.location
+  gi_version = "23.0.0.0"
+  vnet_id = azurerm_virtual_network.example.id
+  license_model = "BringYourOwnLicense"
+  db_servers = [for obj in data.azurerm_oracledatabase_db_servers.example.db_servers : obj.ocid]
+  ssh_public_keys = [ file("~/.ssh/id_rsa.pub") ]
+  display_name = "example-cloud-vm-cluster"
+  cloud_exadata_infrastructure_id = azurerm_oracledatabase_exadata_infrastructure.example.id
+  cpu_core_count = 2
+  hostname = "hostname"
+  subnet_id = azurerm_subnet.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `cloud_exadata_infrastructure_id` - (Required) The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the cloud Exadata infrastructure.
+
+* `cpu_core_count` - (Required) The number of CPU cores enabled on the cloud VM cluster.
+
+* `db_servers` - (Required) The list of DB servers.
```

Comment:
list of? hostnames/ ids? if host names
```suggestion
* `db_server_host_namess` - (Required) The list of DB server hostnames to use.
```

Reply:
@katbyte I agree with your comments on the naming of these arguments, however I have been asked to match the official OCI Terraform provider here (concerns about consistent customer experience), so I don't have a lot of freedom in changing them myself:
https://registry.terraform.io/providers/oracle/oci/latest/docs/data-sources/database_cloud_vm_cluster#db_servers

Reply:
Typo `db_server_host_namess`

Reply:
These changes were not used, so the typo was not committed.